client/app/components/sections/ManageAvailability/ManageAvailability.css

```diff
@@ -54,6 +50,13 @@
   &[disabled]:hover {
     background-color: #eee;
   }
+
+  will-change: background-color;
```

Comment:
Again, only add the `will-change` if you really need it and notice that it makes a difference.